build-logic/jvm/src/main/kotlin/build-logic.test-base.gradle.kts

```diff
@@ -0,0 +1,25 @@
+import com.github.vlsi.gradle.dsl.configureEach
+import org.gradle.api.tasks.testing.Test
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+
+plugins {
+    id("java-library")
+}
+
+tasks.configureEach<Test> {
+    testLogging {
+        exceptionFormat = TestExceptionFormat.FULL
+        showStandardStreams = true
+    }
```

Comment:
please make sure ch.tutteli.gradle.plugins.junitjacoco plugin is applied instead which configures more that this.

Reply:
Well, there's `gradle-extensions-plugin` which implements this formatting: https://github.com/vlsi/vlsi-release-plugins/tree/master/plugins/gradle-extensions-plugin#test-output-formatting

<img src="https://github.com/vlsi/vlsi-release-plugins/raw/master/plugins/gradle-extensions-plugin/github_actions_tests.png">